<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pbConnect">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbDisconnect">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Disconnect</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="vlModeRadios">
         <item>
          <widget class="QFrame" name="frRadios">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <widget class="QRadioButton" name="rbHttp">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>127</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>HTTP</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
           <widget class="QRadioButton" name="rbHttps">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>30</y>
              <width>127</width>
              <height>24</height>
             </rect>
            </property>
            <property name="text">
             <string>HTTPS</string>
            </property>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPlainTextEdit" name="pteRecv"/>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="pteSend">
           <property name="plainText">
            <string>GET / HTTP/1.1
Host: naver.com


</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pbSend">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="tlHost">
         <property name="text">
          <string>Host :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leHost">
         <property name="text">
          <string>naver.com</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="tlHost_2">
         <property name="text">
          <string>Port :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lePort">
         <property name="text">
          <string>80</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
